actor/src/main/resources/reference.conf

```diff
@@ -539,6 +540,19 @@ pekko {
         allow-core-timeout = on
       }
 
+      # This will be used if you have set "executor = "virtual-thread-executor"
+      # This executor will execute the every task on a new virtual thread.
+      # Underlying thread pool implementation is java.util.concurrent.ForkJoinPool for JDK <= 22
+      # If the current runtime does not support virtual thread,
+      # then the executor configured in "fallback" will be used.
+      virtual-thread-executor {
+        #Please set the the underlying pool with system properties below:
+        #jdk.virtualThreadScheduler.parallelism
```

Comment:
can these be supported as configs as opposed to relying on system properties?

Reply:
Yes, but that will need to create a dedicated fork-join pool with a new factory for the `CarrierThread`( but which lives in java.internal.misc`...  we have that internal but not sure if that's a good idea for an opensource project.